src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/BehaviourService.kt

```diff
@@ -56,8 +55,8 @@ class BehaviourService(
     if (lastReviewTime == null) return defaultReviewPeriod()
 
     val today = LocalDateTime.now(clock)
-    return if (Period.between(lastReviewTime.toLocalDate(), today.toLocalDate()).months < DEFAULT_MONTHS) { lastReviewTime } else {
```

Comment:
This is me not understand Period - for things over a year it will return years, months, days to 14 months ago it will return 2 in the month value!